### PR TITLE
Array test

### PR DESCRIPTION
**Pull request by JSchoenbachler**
Adds testing for seeker array functions. Disables fetch tests.

### CODE REVIEW

File: R/fetch.R [hunk 1]
```diff
@@ -120,4 +120,4 @@ fetch = function(
 
   writeLogFile(logPath, n = -length(fs))
   d = list(localFilepaths = getFileVec(localFilepaths), statuses = result)
-  return(d)}
+  return(d)}
```
Comment (jakejh) [inline R/fetch.R]: Add back newline.

File: tests/testthat/test-fetch.R [hunk 2]
```diff
@@ -40,11 +41,15 @@ test_that('fetch', {
...
     list(remoteFilepaths = metadataGSM[[remoteColname]], outputDir = outputDirFetchTest),
     paramsFetchNow))
 
-  for (file in strsplit(result$localFilepaths, ';')[[1]]) {
+  resultExp = snapshot(resultObs, file.path(dataDir, 'fetch_output_testing.qs'))
+
+  expect_equal(resultObs, resultExp)
+
+  for (file in strsplit(resultExp$localFilepaths, ';')[[1]]) {
     file = paste0('./', file)
```
Comment (jakejh) [inline tests/testthat/test-fetch.R]: Why is this line necessary?

**Review by jakejh — CHANGES_REQUESTED** [review]
Similar to previous pull request: go through slowly with a fresh set of eyes and find at least two things to clean up. Set your standards high.

File: tests/testthat/test-fetch.R (hunk 2 above)
Comment (jakejh) [reply tests/testthat/test-fetch.R]: My previous question was regarding paste0.

Comment (JSchoenbachler) [reply tests/testthat/test-fetch.R]: Gotcha, this is now resolved

**Review by JSchoenbachler — COMMENTED** [review]
Resolved issues.

**Review by jakejh — APPROVED** [review]
